api.py

```diff
@@ -615,12 +615,36 @@ def get_article(
     """
     Fetch an individual article record by ID.
     """
+    source = {
+        "includes": [
+            "article_title",
+            "normalized_article_title",
+            "publication_date",
+            "indexed_date",
+            "language",
+            "full_language",
+            "canonical_domain",
+            "url",
+            "normalized_url",
+            "original_url",
+            "text_content",
+        ]
+    }
+    query = {"match": {"_id": id}}
 
     try:
-        hit = ES.get(index=collection.name, id=id)
+        res = ES.search(index=collection.name, source=source, query=query)
+        hits = res["hits"]["hits"]
+        if len(hits) > 0:
+            hit = hits[0]
+        else:
+            raise HTTPException(
+                status_code=404, detail=f"An article with ID {id} not found!"
+            )
```

Comment:
Looks good but I'm not a fan of raising inside try/except. May be:

```python
    try:
        res = ES.search(index=collection.name, source=source, query=query)
        hits = res["hits"]["hits"]
    except TransportError as e:
        raise HTTPException(
            status_code=500,
            detail=f"An error occured when searching for article with ID {id}",
        ) from e

    if hits:
        hit = hits[0]
    else:
        raise HTTPException(
            status_code=404, detail=f"An article with ID {id} not found!"
        )
```

Reply:
My variation on @kilemens
```
    if len(hits) == 0:
        raise HTTPException(
            status_code=404, detail=f"An article with ID {id} not found!"
        )
    hit = hits[0]
```


I suppose another, exceptionalist alternative is:
```
try:
        # expected response format is {"hits": {"hits":[ ... ] }}
        # with an empty list if no matches found, and a single entry on success
        hit = ES.search(index=collection.name, source=source, query=query)["hits"]["hits"][0]
except (TransportError, TypeError, KeyError) as e:
        raise HTTPException(
            status_code=500,
            detail=f"An error occurred when searching for article with ID {id}: {e!r}",
        ) from e
except IndexError:   
        raise HTTPException(
            status_code=404, detail=f"No article with ID {id} found!"
        )
```
I don't doubt ES.search can raise other exceptions as well!